README.md

```diff
@@ -158,6 +158,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 * [infincia/bip39-rs](https://github.com/infincia/bip39-rs) — Rust implementation of BIP39.
 * [Joystream](https://github.com/Joystream/joystream) — A user governed video platform [![Build Status](https://api.travis-ci.org/Joystream/joystream.svg?branch=master)](https://travis-ci.org/Joystream/joystream)
 * [Libra](https://github.com/libra/libra) — Libra’s mission is to enable a simple global currency and financial infrastructure that empowers billions of people.
+* [Lighthouse](https://github.com/sigp/lighthouse) — Rust Ethereum 2.0 Client [![Build Status](https://api.travis-ci.com/sigp/lighthouse.svg?branch=master)](https://travis-ci.com/sigp/lighthouse)
```

Comment:
This refers to a travis build that doesn't seem to actually be active, and the project uses Github actions instead, so could you please add the badge for that instead? Thanks!

Reply:
Thanks for pointing out. Fixed!